app/views/posts/_form.html.erb

```diff
@@ -6,6 +6,21 @@
     <%= form.text_field :body %>
   </div>
 
+  <div>
+    <%= form.label :images, style: "display: block" %>
```

Comment:
Laso you can imporove the label to guidethe user better something like selcet multiple images

Reply:
Done!